<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgInputStart</class>
 <widget class="QDialog" name="dlgInputStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblAction">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Input ...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledInputStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
